petclinic/README.md

```diff
@@ -2,8 +2,17 @@
 The wasmCloud Pet Clinic example application is a WebAssembly and wasmCloud-based reimagining of the classic [Spring Boot microservices Pet
 Clinic](https://github.com/spring-petclinic/spring-petclinic-microservices) example.
 
+The purpose of this README is to help you learn the wasmCloud application runtime. We took great care to describe how we designed the application and how you can extend it. We paid special attention to design concepts and vocabulary. Post questions or concerns about any aspect of this application here... We want to hear your feedback.
+
 ## Application Architecture
-The wasmCloud Pet Clinic consists of the following five [actors](./actors):
+The wasmCloud Pet Clinic application has the following simple architecture: 
```

Comment:
The changes you've made are a great improvement to this README. 
One part feels a bit out of place: the Application Architecture section now begins with a sequence of operations that aren't architecture. These lines (Start all five ..., Push ..., Ensure ...) could go into the front of the section on Running, with an intro sentence something like "The process of starting an actor includes these steps: "  (then the lines you've added). The "run.sh" script automates these steps for you.

After moving those lines, the Architecture section would begin with "The wasmCloud Pet Clinic consists ...."

Reply:
I agree with Steve here and I think this section would go well below the architecture steps